packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-data.tsx

```diff
@@ -58,6 +71,10 @@ const MatchData = () => {
 						...data.match,
 						firstTeam: data.match.firstTeam,
 						secondTeam: data.match.secondTeam,
+						tournament: {
```

Comment:
Much magic for me in the code written in this file haha! Would love if someone explained to me in a point in time :D

Reply:
I think I'll speak to you about this instead :P